contrib/babelfishpg_tsql/src/iterative_exec.c

```diff
@@ -1087,6 +1087,20 @@ is_xact_abort_on_error(PLtsql_execstate *estate)
 	return false;
 }
 
+/*
+ * For Unmapped PostgreSQL errors: ERRCODE_UNDEFINED_COLUMN and ERRCODE_UNDEFINED_TABLE, 
+ * we will allow it to go to CATCH block of a TRY CATCH STATEMENT, if error is raised from lower level of execution
+ */
+static
+bool
+escape_last_unmapped_error(PLtsql_execstate *estate)
+{
+	if (!is_error_raising_batch(estate) && last_error_mapping_failed && (latest_pg_error_code == ERRCODE_UNDEFINED_TABLE || latest_pg_error_code == ERRCODE_UNDEFINED_COLUMN))
+		return true;
+	
+	return false;
+}
+
```

Comment:
What is the expected behaviour if this error was raised inside a plpgsql procedure. Currently we do no catch any errors coming from plpgsql in a pltsql catch block, except for system functions/proc in sys/pg_catalog.

Reply:
Also lets add some more information in the PR description about how this error is being handled (transaction behaviour, interoperability, etc).

Reply:
Added tests on plpgsql procedures, refer: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/341ce31307fadf35557de31560ffc454b765abe2/test/JDBC/expected/BABEL-5186-vu-verify.out#L4081